cmd/snapctl/task.go

```diff
@@ -477,22 +481,39 @@ func listTask(ctx *cli.Context) error {
 		"LAST FAILURE",
 	)
 	for _, task := range tasks.ScheduledTasks {
+		lastErrorMessage := task.LastFailureMessage
```

Comment:
I don't think this is needed anymore. Probably makes sense to just pass task.LastFailureMessage in directly.